### PR TITLE
Support vibe-core

### PR DESCRIPTION
**Pull request by svandenhoek**
- [ ] Functionality works & meets specs
- [x] Code reviewed
- [x] Documentation was updated

After merge:
- [ ] Added feature/fix to draft release notes

### CODE REVIEW

File: molgenis-app-vibe/src/main/java/org/molgenis/vibe/VibeController.java [hunk 1]
```diff
@@ -31,6 +39,14 @@
   private final VibeJobExecutionFactory vibeJobExecutionFactory;
   private final DataService dataService;
   private final FileStore fileStore;
+  private static final Gson gson;
+
+  static {
```
Comment (dennishendriksen) [inline molgenis-app-vibe/src/main/java/org/molgenis/vibe/VibeController.java:44]: Code duplication with VibeServiceImpl

File: molgenis-app-vibe/src/main/java/org/molgenis/vibe/VibeServiceImpl.java [hunk 2]
```diff
@@ -3,29 +3,43 @@
...
 import org.springframework.stereotype.Service;
 
 @Service
 class VibeServiceImpl implements VibeService {
   private final DataService dataService;
   private final FileStore fileStore;
   private final FileMetaFactory fileMetaFactory;
+  private static final Gson gson;
+
+  static {
```
Comment (dennishendriksen) [inline molgenis-app-vibe/src/main/java/org/molgenis/vibe/VibeServiceImpl.java:37]: Code duplication with VibeController

File: molgenis-app-vibe/src/main/java/org/molgenis/vibe/VibeController.java (hunk 1 above)
Comment (svandenhoek) [reply molgenis-app-vibe/src/main/java/org/molgenis/vibe/VibeController.java:44]: Noted, fix will be added.

File: molgenis-app-vibe/src/main/java/org/molgenis/vibe/VibeServiceImpl.java (hunk 2 above)
Comment (svandenhoek) [reply molgenis-app-vibe/src/main/java/org/molgenis/vibe/VibeServiceImpl.java:37]: Noted. Fix will be added.

**Comment by svandenhoek** [comment]
Based on feedback the gson (de)serializing is now moved to a separate class. However, it might be better to move this class to `vibe-core`. So keep it here for now so this pull-request can be closed or make a new `vibe-core` release instead?

File: molgenis-app-vibe/pom.xml [hunk 3]
```diff
@@ -135,7 +135,7 @@
   <dependencies>
     <dependency>
       <groupId>org.molgenis</groupId>
-      <artifactId>vibe</artifactId>
+      <artifactId>vibe-core</artifactId>
```
Comment (dennishendriksen) [inline molgenis-app-vibe/pom.xml:138]: Are the exclusions still required?

Comment (dennishendriksen) [reply molgenis-app-vibe/pom.xml:138]: will be resolved in new vibe-core version. @svandenhoek can you create an issue in vibe repo?

Comment (svandenhoek) [reply molgenis-app-vibe/pom.xml:138]: Done: https://github.com/molgenis/vibe/issues/57

**Pull request merged**